ansible/roles/software/ckan/catalog/ckan-app/defaults/main.yml

```diff
@@ -86,3 +86,5 @@ catalog_ckan_fgdc2iso_host: localhost
 ckan_catalog_next: false
 ckan_production_ini_template: etc_ckan_production.ini.j2
 ckan_uses_gunicorn: false
+newrelic_license_key: some-secret
```

Comment:
Secrets should not have defaults. Instead, assume it is required and add an `assert` that the variable is defined at the beginning of the ckan-app role.